src/components/layout/index.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+import './layout.scss'
+
+const Layout = ({
+  children,
+  className = '',
+  tag: Tag = 'div',
+  ...restProps
+}) => (
+  <Tag className={`app-container ${className}`} {...restProps}>
```

Comment:
Component is self-contained. Styles that are written inside the component are contained and may be hard-coded. Although, I allow to redefine actual `tag` and append class names from above.

Here `.app-container` is a part of component and is hard-coded because of this.

Does this make sense?

Reply:
In this case, I wanted to create a CSS file to put styles into but I did not know where. So I created a small component for it.

If you can suggest something better, I am open for it.

Reply:
No it's ok this way. Thanks